src/web/handlers.rs

```diff
@@ -538,21 +538,10 @@ pub async fn heartbeat(hb: HeartbeatRequest) -> Result<HttpResponse, Error> {
 // try returning an API error
 pub async fn test_error(
     _req: HttpRequest,
-    ter: TestErrorRequest,
+    _ter: TestErrorRequest,
 ) -> Result<HttpResponse, ApiError> {
     // generate an error for sentry.
 
-    /*  The various error log macros only can take a string.
-        Content of Tags struct can be logged as KV (key value) pairs after a `;`.
-        e.g.
-        ```
-        error!("Something Bad {:?}", err; wtags)
-        ```
-
-        TODO: find some way to transform Tags into error::KV
-    */
-    error!("Test Error: {:?}", &ter.tags);
```

Comment:
maybe worth keeping the error! statement? I think actix dumps the error out somewhere (but not to logs) and we always tend to log something to error during `/__error__`